src/code/z_elf_message.c

```diff
@@ -47,7 +47,7 @@ u32 ElfMessage_CheckCondition(ElfMessage* msg) {
 
     // "Unplanned conditions"
     LOG_STRING("企画外 条件", "../z_elf_message.c", 156);
-    __assert("0", "../z_elf_message.c", 157);
+    ASSERT(false, "0", "../z_elf_message.c", 157);
```

Comment:
I think this kind of assert just be `ASSERT(0)` like it was originally
```suggestion
    ASSERT(0, "0", "../z_elf_message.c", 157);
```